source/protobuf/mapdata.proto

```
@@ -0,0 +1,53 @@
+syntax = "proto2";
+
+package clienteditor.protobuf.mapdata;
```

Buf (1.65.0)

[error] Files with package "clienteditor.protobuf.mapdata" must be within a directory "clienteditor/protobuf/mapdata" relative to root but were in directory "source/protobuf".

(PACKAGE_DIRECTORY_MATCH)